### PR TITLE
Remove force-version flag when calling starting score indexer

### PR DESCRIPTION
**Pull request by notbakaneko**
We're looking at remove the `--force-version` flag from the score indexer ppy/osu-elastic-indexer#137; github actions can just set the version with `run` before the tests.

~~This still has the issue for with docker compose where the score index isn't aliased after the docker containers start and has to be done separately (which is what `force-version` got around)~~

Tries to set the score indexer schema version when a fresh migration is run so the score indexer can setup the alias for it.
Uses a loop in case the score indexer hasn't finished starting yet - the alternative is `healthcheck` on score-indexer which needs more things installed into the image as well as parsing the output, so I just went with adding it to the migration command.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -141,6 +141,7 @@ jobs:
 
       - name: Setup indices
         run: |
+          php artisan es:index-scores:set-schema --schema=1
```
Comment (nanaya) [inline .github/workflows/tests.yml:144]: the actual schema version should be defined using env instead so it always matches docker compose

**Review by nanaya — CHANGES_REQUESTED** [review]
adding set schema command in docker dev prepare script is probably fine

**Comment by peppy** [comment]
@notbakaneko was this intended to address the issue mentioned in https://github.com/ppy/osu-elastic-indexer/pull/137#issuecomment-1404973857? If so can we revisit it? Would like to cross that PR off the list.

**Comment by notbakaneko** [comment]
I remember running into an issue on the indexer side when testing the changes and didn't get further than that yet

**Comment by notbakaneko** [comment]
`set-schema` needs to be after the indexer starts in order to alias any newly created index correctly

**Comment by nanaya** [comment]
So what to do with dev env here?

**Comment by nanaya** [comment]
is it just me or keeping `--force-version` would be way simpler?

**Comment by notbakaneko** [comment]
All the additional stuff isn't needed
#10477

**Pull request closed without merging**